src/components/PostSection/index.jsx

```diff
@@ -112,6 +131,9 @@ const PostSection = ({
         {ownedByCurrentUser && (
           <OptionsBtn options={options} position="left top" />
         )}
+        {!ownedByCurrentUser && (
+          <OptionsBtn options={optionReport} position="left top" />
+        )}
```

Comment:
You just send report options, so there is no delete button in the popup

Reply:
yeap it's not supposed for anyone to see the delete option except the owner